system/vrops-custom-importer/README.md

```diff
@@ -0,0 +1,8 @@
+vrops-custom-importer
+--------------
+
+Installs the [vrops-custom-importer](https://github.tools.sap/cia/vrops-custom-importer) operator which import custom VMWare metrics to vrops.
+
+## Configuration
+
+See [values](./values.yaml).
```

Comment:
A little bit more explanation would be good, like creds for vcenter and vrops are needed. Maybe a picture, what is pushing and fetching from where to where.

Reply:
ok probably this could go into the source repository. but still some more explanation would be cool, what are the crucial things to specify.